### PR TITLE
docs: Link to source proto; improve Publishing

### PR DESCRIPTION
**Pull request by lorensr**

### CODE REVIEW

File: packages/proto/lib/index.d.ts [hunk 1]
```diff
@@ -20,8 +22,13 @@
  * import { temporal } from '@temporalio/proto';
  * const retryPolicy: temporal.api.common.v1.IRetryPolicy = { ... };
  * ```
+ * 
+ * The source protos are in the [api repo](https://github.com/temporalio/api/), for example [`RetryPolicy` in `temporal/api/common/v1/message.proto`](https://github.com/temporalio/api/blob/eff34dc3a970cdbf3a10a7be827edf1b1528b125/temporal/api/common/v1/message.proto#L96).
```
Comment (bergundy) [inline packages/proto/lib/index.d.ts:26]: Some of the protos are there and some in sdk-core, we take the API protos from a subtree in Core

**Review by bergundy — COMMENTED** [review]
Just one comment about the modified docstring

File: packages/proto/lib/index.d.ts (hunk 1 above)
Comment (lorensr) [reply packages/proto/lib/index.d.ts:26]: Some of the `temporal` protos are in sdk-core?

Comment (bergundy) [reply packages/proto/lib/index.d.ts:26]: They're in a git subtree there, the original is in the API repo

Comment (lorensr) [reply packages/proto/lib/index.d.ts]: Updated to use the sdk-core repo version of temporalio/api.

Comment (lorensr) [reply packages/proto/lib/index.d.ts]: The sdk-core protos I link to earlier in this file

**Review by bergundy — APPROVED** [review]

**Pull request merged**